### ISSUE TITLE
Chapter5 executable

### ISSUE THREAD

**Issue by esel-fliegen** [PacktPublishing/Building-Computer-Vision-Projects-with-OpenCV4-and-CPlusPlus#4]
I think the executable instruction for Chapter5 should be, "./Chapter5 ../data/test.pgm" when inside the build dir. That's how I got it to work.

**Comment by damiles** [comment]
yes, in the documentation is not set the full path to segment, but you can choose any of the tree images that are in the data path. I will add a bit more of extra info in the readme to be more clear.
